src/Components/Card.jsx

```diff
@@ -0,0 +1,40 @@
+import "./Styles/Card.css";
+
+import React from "react";
+
+export default function Card({ Item }) {
+
+    return (
+        <>
+            <div class="instafresh-card card card-with-badge card-with-text-overlay">
+
+                <div class="card-img flex-justify-center">
+                    <img class="card-img flex-justify-center" src={Item.imageUrl} alt="Product" />
+                </div>
+
+                <div class="card-title item-price-detail">
+                    <h3>{Item.name}</h3>
+                    <p>₹{Item.price}  {!(Item.discount === 0) ? <s class="text-s">₹{Item.prevPrice}</s> : <s></s>}</p>
+                </div>
+
+                <div className="discount-badge">
+                    {!(Item.discount === 0) ? <div class="discount special-badge text-overlay">{Item.discount}% off</div> : <div></div>}
```

Comment:
Dont render empty div's. you can pass `null`

Reply:
Thank you Tahir for pointing this out. Changed it in latest commit.